tests/shared.py

```
@@ -118,26 +118,30 @@ def get_driver_features(backend):
         response = backend.sendAndReceive(protocol.GetFeatures())
         if not isinstance(response, protocol.FeatureList):
             raise Exception("Response is not instance of FeatureList")
-        features = set(response.features)
+        raw_features = set(response.features)
+        features = set()
+        for raw in raw_features:
+            try:
+                features.add(protocol.Feature(raw))
+            except ValueError:  # ignore features we don't know
+                warnings.warn('Unknown feature "%s"' % raw)
```

We should fail hard on it. 

It makes easier to stop if we have a typo, for example.